pkg/tfsandbox/tf_file.go

```diff
@@ -42,6 +49,150 @@ func (l *locals) createLocal(v interface{}) string {
 	return key
 }
 
+type mapper struct {
```

Comment:
nit: could we name this mapper according to what it maps? It looks like it maps pulumi Properties or Types.
(I had a bit of overload whiplash from converter mappings reading through this)